docs/source-pytorch/common/optimization.rst

```diff
@@ -83,113 +63,6 @@ Gradient Accumulation
 .. include:: ../common/gradient_accumulation.rst
 
 
-Use Multiple Optimizers (like GANs)
```

Comment:
The same section already exists in manual_optimization.rst below

Reply:
can we just add cross-link?

Reply:
done

Reply:
unfortunately didn't work, because the page is included via .. include and then the label is duplicated. A limitation of include directive.